### DISCUSSION TITLE
Ditto release 3.3.0 planning

### DISCUSSION THREAD

**Discussion by thjaeckle** [eclipse-ditto/ditto#1660] [General]
Hi community and @eclipse-ditto/iot-ditto-committers 

It has already been 3 months since the last Ditto release, so let's discuss on the upcoming Ditto minor release, 3.3.0.
The issues in it can be seen here: https://github.com/orgs/eclipse-ditto/projects/1/views/2

Namely:
* #1593
* #1614 
* #1524 
* #1623 
* #1630 
* #1635 
* #1641 
* #1644 
* #1658 

Some PRs are still waiting for approval, would be good to get those ;)

In my opinion that is again enough for a minor release. Good features and improvements contained ..
What do you think?
Can we plan the release for lets say next week?

**Comment by kalinkostashki** [comment]
@thjaeckle Sure, I will check the issue that are left the review and run the system tests.
Other than that I see no problems with doing the release next week :)
